README.md

```diff
@@ -1 +1,41 @@
-# Leadboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard: setup project
+
+> In this activity I set up an enivoroment using webpack  and worote basic html with minimum styling for the Leaderboard list app.
+
+![leaderboard](https://user-images.githubusercontent.com/92645881/156154982-6ef410fe-8ec8-46fd-af85-2576bc8a197c.png)
+
+## Built With
+
+- `HTML` ,`CSS` and `JavaScript`
+- Frameworks : `Webpack`
+- Technologies used: `GitHub`
+
+
```

Comment:
- [ ] Kindly add some steps on how to use your project locally so your project is documented professionally.